package.json

```diff
@@ -67,5 +67,10 @@
     "posttest": "eslint . --ext .ts --config .eslintrc",
     "prepack": "rm -rf lib && tsc -b && oclif manifest && oclif readme",
     "version": "oclif readme && git add README.md"
+  },
+  "csdxConfig": {
+    "shortCommandName": {
```

Comment:
@abhinav-from-contentstack Can you list down all the commands in this plugin and add their short names?